types/draft-js/index.d.ts

```diff
@@ -920,7 +923,7 @@ declare namespace Draft {
 
             class RichTextEditorUtil {
                 static currentBlockContainsLink(editorState: EditorState): boolean;
-                static getCurrentBlockType(editorState: EditorState): DraftBlockType;
+                static getCurrentBlockType<CustomBlockType = void>(editorState: EditorState): DraftBlockType<CustomBlockType>;
```

Comment:
`RichTextEditorUtil.getCurrentBlockType` can return any string. So, even if you write this:

```ts
const blockType = RichTextEditorUtil.getCurrentBlockType(editorState)
```

The type of `blockType` isn't guaranteed to be `CoreDraftBlockType` (but your typings suggests so). It can be any string (and you can't change the implementation of that function). So, your typings is actually be wrong.

Similar things can be said for other changes too.